src/components/screensavermanager.js

```diff
@@ -18,6 +20,42 @@ define(["events", "playbackManager", "pluginManager"], function (events, playbac
         }
     });
 
+    function getScreensaverPlugin() {
+
+        var apiClient = connectionManager.currentApiClient();
+
+        if (apiClient && apiClient.isLoggedIn()) {
+            isLoggedIn = true;
+        }
+        var option = function () {
+            try {
+                return userSettings.get("screensaver", false);
+            } catch (err) {
+                return null;
+            }
+        }();
+
+        var defaultOption = function (isLoggedIn) {
+            if (isLoggedIn) {
+                return "backdropscreensaver";
+            }
+
+            return "logoscreensaver";
+        }(isLoggedIn);
+
+        if ((option = option || defaultOption) === "none") {
```

Comment:
what this `option = option` does? look like a hack, maybe just
```js
if (option === "none" || defaultOption === "none") {
```

Reply:
https://github.com/jellyfin/jellyfin-web/pull/592/commits/f28074fb9ed25b6566c0dd58ee8b3ccec96e4adf